manual/src/cmds/runtime-tracing.etex

```diff
@@ -369,7 +369,7 @@ they need to:
 Note that if the emitter and the consumer are different programs, both must
 perform steps 1 to 3 to register custom events and custom event types (if any).
 Note that the tag values need not be the same in both programs; the only value
-that should be identical the same is the name.
+that should match are the names.
```

Comment:
the only values that should match (apologies for not catching this earlier)

Reply:
My bad. 773fbc4